saleor/graphql/views.py

```diff
@@ -190,10 +190,14 @@ def format_error(error):
             result = format_graphql_error(error)
         else:
             result = {"message": str(error)}
+
+        exc = error
+        while isinstance(exc, GraphQLError) and hasattr(exc, "original_error"):
+            exc = exc.original_error
+
+        traceback.print_exception(type(exc), exc, exc.__traceback__)
```

Comment:
This should really be `logging.error(..., exc_info=exc)`. In production environments, no one can hear you scream.

Reply:
Thanks! That's what I wanted to do, but it didn't work this time. I always though it was `logger.exception(exc)`.

It works in my other projects somehow... I will have to update that.

Thanks again.

Reply:
`logger.exception` accepts a message as its first argument, it will cast whatever you pass it to a string. That's not where it gets the exception info from. Underneath it does something like this:

```python
import sys


def exception(self, msg, **kwargs):
    exc_info = sys.exc_info()
    return self.error(msg, **kwargs, exc_info=exc_info)
```

However `sys.exc_info()` only works if you're currently inside an `except:` block.